### DISCUSSION TITLE
Trip time (PID) with Unit Conversion

### DISCUSSION THREAD

**Discussion by guirosasilva** [agronick/aa-torque#154] [General]
Good morning,

I was trying to show the Trip Time using the ready formula Second to Minutes:Seconds, but the display doesn't show any results. I don't know if it's because the decimal places requested, or I'm doing anything wrong. If I do edit the formula to for example (a /60), it's demonstrated correct minutes passed by, but the decimal place obviously is like a minute part, but not in seconds.
![Screenshot_20240927_200056_Torque](https://github.com/user-attachments/assets/0ee62655-7d32-470a-a7bf-9db7af9267fa)
On Torque app, it's showing in "needed" form. Sorry about my poor English.

**Comment by agronick** [comment]
In unit conversions / formulas look at Seconds to Minutes:Seconds.
